README.md

```diff
@@ -1,3 +1,14 @@
+<details>
+  <summary>FYI: Updating to v0.4.0</summary>
+  
+  - `answer_str` -> `answer`
+  - Don't write `.send()` after each request, now it is done automatically.
+  - Use `.auto_send()` to construct your bot: `let bot = Bot::from_env().auto_send();`.
+  - `UpdateWithCx<Message>` -> `UpdateWithCx<AutoSend<Bot>, Message>`
```

Comment:
These three should be merged somehow, so users understand that it's `auto_send` that allows omitting `send` and requires writing `AutoSend<Bot>`.